src/models/linear_regression.jl

```diff
@@ -15,23 +16,22 @@ best for numerical reasons.
 - `multi_output`: for maximum efficiency, specify `multi_output=true/false` """
 function LinearRegression(; eltype=Float64, multi_output=nothing)
     if multi_output === nothing
-        LinearRegression{Array{eltype}}()
-    elseif multi_ouput::Bool
-        LinearRegression{Array{eltype}, 2}()
-    else
-        LinearRegression{Array{eltype}, 1}()
-    end        
+        LinearRegression{Array{eltype}, eltype}()
+    elseif multi_output::Bool != nothing
```

Comment:
That line looks a bit weird. The first `if` checks if `multi_output === nothing`, so there's no reason to check if it's `!= nothing` at this point. Did you mean to do something else?

Reply:
I meant to guarantee that regardless of the value passed to the parameter `multi_output` we would always parametrize the LinearRegression model with `Array{eltype, 2}`. This is necessary because of the line  `lr.coefs = results[2:end,:];` which always returns an `Array{eltype, 2}` .

I agree that this is not the best approach. Maybe we could instead test if the `multi_output` isa `Boolean` value?